### PR TITLE
feat: use the latest Spectral

### LINKED ISSUE #107: Update to spectral@6
**Chore summary**
Spectral just got a lot of new features with version 6 including a new ruleset format, better errors, overrides and aliases.

**Tasks**
- [ ] Update to use Spectral V6

### LINKED ISSUE #132: Update to the latest version of spectral
**Chore summary**
Currently this extension is on v5 of spectral. The latest version of Spectral is [6.3.0](https://www.npmjs.com/package/@stoplight/spectral-cli). Let's update it to use the latest. 

**Tasks**
- [ ] Bump spectral version to 6.3.0
- [ ] Test that it still works 🚀

### PR DESCRIPTION
**Pull request by P0lip**
Closes #132
Closes #107 

**Checklist**

- [x] Tests added / updated
- [ ] Docs added / updated

**Does this PR introduce a breaking change?**

- [x] Yes
- [ ] No

Testing steps:
- in VS Code: remove the existing extension
- In Repo: `./make.js allDev`
-  in VS Code: go to extensions and click "Install from VSIX" image
- in VS Code: select the built artifact from the `artifacts` directory
-  in VS Code: use the extension :tada:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.2.5] - 2020-02-01
+## [1.0.0] - 2022-04-25
+
+### Features
+
+- Updated to the latest version of Spectral
```
Comment (lottamus) [inline CHANGELOG.md:12]: Should we list which version? Or maybe that doesn't matter? I noticed we did that in the past

```
- Update to Spectral vX.X.X
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.2.5] - 2020-02-01
+## [1.0.0] - 2022-04-25
+
+### Features
+
+- Updated to the latest version of Spectral 
+- Changes made to custom functions are now recognized by the extension, and the ruleset is updated accordingly
```
Comment (lottamus) [inline CHANGELOG.md:13]:
```suggestion
- Changes made to custom functions are now recognized by the extension, and the ruleset is updated automatically
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,7 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.2.5] - 2020-02-01
+## [1.0.0] - 2022-04-25
```
Comment (lottamus) [inline CHANGELOG.md:8]: Are we waiting to release it or doing it today?

```suggestion
## [1.0.0] - 2022-04-21
```

**Review by lottamus — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (P0lip) [reply CHANGELOG.md:8]: yeah wasn't sure when we'd merge it so I put Monday

**Comment by g-borges** [comment]
@P0lip -  this new release seems to have failed. Last version published on Marketplace is [0.2.5](https://marketplace.visualstudio.com/items?itemName=stoplight.spectral&ssr=false#version-history). [Release workflow](https://github.com/stoplightio/vscode-spectral/runs/6115922750?check_suite_focus=true) failed to run `vsce publish`.

**Comment by P0lip** [comment]
Yup, we're aware. We're working on it.
I'll let you know when it's released.

**Pull request merged**